docs/windows.md

```diff
@@ -0,0 +1,195 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports to running on Windows worker Node, and setups an overlay network
+which uses VXLAN tunneling to route packets between nodes.
```

Comment:
We support only VXLAN on Windows?

Reply:
We can support VXLAN and STT tunnels on windows. Add `STT` in doc.